platforms/hyperledger-indy/charts/indy-node/values.yaml

```diff
@@ -25,6 +25,10 @@ organization:
   #Eg. name: provider
   name:
 
+genesis:
+  pool:
```

Comment:
why is "type" missing here. (Should be replaced by add_org)
Also description of why this parameter is needed is missing.